hidimstat/Dnn_learner.py

```diff
@@ -1,66 +1,65 @@
 import numpy as np
 from sklearn.base import BaseEstimator
 
-from .Dnn_learner_single import DNN_learner_single
-from .utils import sigmoid, softmax
+from .Dnn_learner_single import Dnn_learner_single
 
 
-class DNN_learner(BaseEstimator):
-    """ToDo
+class Dnn_learner(BaseEstimator):
```

Comment:
```suggestion
class DnnLearner(BaseEstimator):
``` Class names are CamelCase